suite-common/suite-types/src/device.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    DeviceButtonRequest,
+    DeviceEvent,
+    KnownDevice,
+    PROTO,
+    UI,
+    UnknownDevice as UnknownDeviceBase,
+    UnreadableDevice as UnreadableDeviceBase,
+} from '@trezor/connect';
+import { DeviceMetadata } from '@suite-common/metadata-types';
+
+// TODO former type was from suite Action['type'][] - check if this is correct
+const blockedActions: string[] = [UI.CLOSE_UI_WINDOW];
```

Comment:
check this

Reply:
I would prefer some type redux-toolkit or if notthing suitable here, at least alias like `type ActionType = string`